lib/public/utilities/ObservableData.js

```diff
@@ -129,9 +146,15 @@ class ObservableDataBuilder {
      * @return {ObservableData} the build observable data
      */
     build() {
-        const observableData = new ObservableDataWithProcessing(this._initialValue, this._processors);
+        const initialValue = this._initialValue ?? this._source?.getCurrent();
+        const observableData = new ObservableDataWithProcessing(initialValue, this._processors);
+        if (this._source) {
+            const source = this._source;
+            source?.observe(() => observableData.setCurrent(source.getCurrent.bind(source)()));
```

Comment:
```suggestion
            source.observe(() => observableData.setCurrent(source.getCurrent()));
```

Reply:
Removed the spurious `?` but the bind is mandatory here

Reply:
Removed, you are right